src/com/sun/ts/tests/servlet/api/jakarta_servlet_http/sessioncookieconfig/TestListener.java

```diff
@@ -47,48 +47,48 @@ public void contextInitialized(ServletContextEvent sce) {
     scf.setSecure(isSecure);
 
     if (!scf.getComment().equals(comment.toString())) {
-      comment.append("|getCommentFAILED-expecting-" + comment + "-got-"
+      comment.append("|getComment-FAILED-expecting-" + comment + "-got-"
           + scf.getComment());
       scf.setComment(comment.toString());
     }
 
     if (!scf.getPath().equals(path)) {
       comment
-          .append("|getPathFAILED-expecting-" + path + "-got-" + scf.getPath());
+          .append("|getPath-FAILED-expecting-" + path + "-got-" + scf.getPath());
       scf.setComment(comment.toString());
     }
 
     if (!scf.isSecure()) {
       comment.append(
-          "|isSecureFAILED-expecting-" + isSecure + "-got-" + scf.isSecure());
+          "|isSecure-FAILED-expecting-" + isSecure + "-got-" + scf.isSecure());
       scf.setComment(comment.toString());
     }
     if (scf.isHttpOnly()) {
-      comment.append("|isHttpOnlyFAILED-expecting-" + httpOnly + "-got-"
+      comment.append("|isHttpOnly-FAILED-expecting-" + httpOnly + "-got-"
           + scf.isHttpOnly());
       scf.setComment(comment.toString());
     }
     if (!scf.getDomain().equals(domain.toString())) {
       comment.append(
-          "|getDomainFAILED-expecting-" + domain + "-got-" + scf.getDomain());
+          "|getDomain-FAILED-expecting-" + domain + "-got-" + scf.getDomain());
       scf.setComment(comment.toString());
     }
 
     if (scf.getMaxAge() != maxage) {
       comment.append(
-          "|getMaxAgeFAILED-expecting-" + maxage + "-got-" + scf.getMaxAge());
+          "|getMaxAge-FAILED-expecting-" + maxage + "-got-" + scf.getMaxAge());
       scf.setComment(comment.toString());
     }
 
     if (scf.getName() != null) {
```

Comment:
We should allow for a container to a pre configured the name by calling `setName("JSESSIONID")`
```suggestion
    if (scf.getName() != null && !"JSESSIONID".equals(scf.getName()) {
```